src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -441,17 +448,54 @@ export class GwtCallback extends EventEmitter {
       GwtCallback.unimpl('desktop_open_terminal');
     });
 
-    ipcMain.handle('desktop_get_fixed_width_font_list', () => {
-      GwtCallback.unimpl('desktop_get_fixed_width_font_list');
-      return '';
+    ipcMain.on('desktop_get_fixed_width_font_list', (event) => {
+      event.returnValue = monospaceFonts.join('\n');
     });
 
-    ipcMain.handle('desktop_get_fixed_width_font', () => {
-      GwtCallback.unimpl('desktop_get_fixed_width_font');
-      return '';
+    ipcMain.on('desktop_get_fixed_width_font', (event) => {
+      // TODO: Read user preference for font
+
+      let defaultFonts: string[];
+      if (process.platform === 'darwin') {
+        defaultFonts = ['Monaco'];
```

Comment:
Nit: the default monospace font on macOS is Menlo nowadays; is it worth changing that here?

Reply:
I did not realize that! It's probably worth changing and I gave it a quick try. It still defaulted to Monaco because the style was overridden by Ace's default style. I'll have to look at what needs to be done to fix this. I think it needs a call to `preview_.setFont` since I don't see that being done anywhere when the appearance preferences pane is created.

FYI: This is also an issue in Prairie Trillium. The selected font in the dropdown is not the one used in the preview until the selection is changed.